bin/lib/queue.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+class Queue {
+	constructor (items) {
+		this.items = items || [];
+	}
+
+	attachDirectlyInstance (directly) {
+		this.directly = directly;
+	}
+
+	push (func) {
+		this.items.push.apply(this.items, [].slice.call(arguments));
```

Comment:
[`Array.from()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) is another way of doing array-things to array-like-things, or could maybe do this? Unless I'm misreading:

```js
push(funcs) {
  this.items.concat(funcs)
}
```

Reply:
something for Rhys...